specs/core/0_beacon-chain.md

```diff
@@ -1107,7 +1107,7 @@ def get_total_balance(state: BeaconState, validators: List[ValidatorIndex]) -> G
 
 ```python
 def get_fork_version(fork: Fork,
-                     epoch: Epoch) -> int:
+                     epoch: Epoch) -> bytes4:
```

Comment:
Ah sorry, one sec, we haven't defined `Bytes4` custom type yet, so either changed this line to
```suggestion
                     epoch: Epoch) -> bytes:
```
Or defined `Bytes4` and then here we can do:
```suggestion
                     epoch: Epoch) -> Bytes4:
```

What do you think about it? :)

Reply:
`bytesN` (hence `bytes4`) is defined [here](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/simple-serialize.md#aliases). Defining `Bytes4` also is fine I guess.

Reply:
`bytes4` isn't valid python outside of the string context unless we define a new type in custom types. I think I'd rather just leave this as `bytes`

Reply:
Done! Also makes the diff super small.